installer/x86_64/install.sh

```diff
@@ -485,7 +514,14 @@ fi
 
 # Add a menu entry for the DEMO OS
 # Note: assume that apparmor is supported in the kernel
-demo_grub_entry="$demo_volume_revision_label"
+demo_grub_entry="$demo_volume_revision_label-${git_revision}"
+if [ "$environment" = "sonic" ]; then
+    running_sonic_revision=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
```

Comment:
You're correct. Thanks!

---
In reply to: [109413786](https://github.com/Azure/sonic-buildimage/pull/464#discussion_r109413786) [](ancestors = 109413786)